openslides_backend/action/motion/update_metadata.py

```diff
@@ -0,0 +1,32 @@
+import time
+
+from ...models.models import Motion
+from ..base import ActionPayload, DataSet
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("motion.update_metadata")
+class MotionUpdateMetadata(UpdateAction):
+    """
+    Action to update motion metadata.
+    """
+
+    model = Motion()
+    schema = DefaultSchema(Motion()).get_update_schema(
+        properties=[
+            "state_extension",
+            "recommendation_extension",
+            "category_id",
+            "block_id",
+            "supporter_ids",
+            "tag_ids",
+            "attachment_ids",
+        ]
+    )
+
+    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
```

Comment:
Why don't you use `update_instance` here?

Reply:
This is old code, of the existing update_metadata, I did not change it.

Reply:
Changed to update_instance